### DISCUSSION TITLE
iFit Rower compatibility/capability

### DISCUSSION THREAD

**Discussion by rbolinger** [cagnulein/qdomyos-zwift#1425] [Ideas]
I have managed to get my WaterRower S4 with COM Module rower working with the QZ app, thanks for the help on that Roberto. Next step is getting that connected to the iFit app I have running on an android tablet on the rower. I managed to pair to the iFit app using the Virtual iFit option and having my QZ device named I_EL, when it connects the iFit app switches over to Bike and the QZ app crashes.

Please add the iFit compatibility for rowers, I'm happy to test it out. Love iFit but they simply don't have a good rower that works natively, I think a decent number of people will use this and I'll make sure to post something on the WaterRower and iFit Reddit communities once this is working for me.

**Comment by cagnulein** [comment]
Ok I'm converting this on a feature request!
